doc/classes/Node.xml

```diff
@@ -438,7 +438,30 @@
 			<return type="void">
 			</return>
 			<description>
-				Prints the scene to stdout. Used mainly for debugging purposes.
+				Prints the tree to stdout. Used mainly for debugging purposes. This version displays the path relative to the current node, and is good for copy/pasting into the get_node() function.
+				[code]
```

Comment:
You should use `[codeblock]` for code blocks, and indent the block to match the rest of the `<description>`. If you need further indentation in the code block itself (like for GDScript functions, use 4 spaces per indent).